src/server/api/transactions/queries.ts

```
@@ -0,0 +1,329 @@
+    accountId?: string;
+    /** The ID of the subaccount to filter by */
+    subaccountId?: string;
+    /** The start date of the date range to filter by */
+    fromDate?: string;
+    /** The end date of the date range to filter by */
+    toDate?: string;
+    /** The page number to fetch */
+    page?: number;
+    /** The number of transactions per page */
+    pageSize?: number;
+  } | void,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)